RubeesEat.UnitTests/Model/InMemoryPersonRepositoryTests.cs

```diff
@@ -0,0 +1,61 @@
+using System.Collections.Immutable;
+using RubeesEat.Model;
+
+namespace RubeesEat.UnitTests.Model;
+
+[TestFixture]
+public class InMemoryPersonRepositoryTests
+{
+    [Test]
+    public void GetAll_EntriesInDictionary_ReturnsEntries()
+    {
+        var storage = new Dictionary<Guid, Person>();
+        storage.Add(TestDomain.TestPersons[0].Id, TestDomain.TestPersons[0]);
+        storage.Add(TestDomain.TestPersons[1].Id, TestDomain.TestPersons[1]);
+        storage.Add(TestDomain.TestPersons[2].Id, TestDomain.TestPersons[2]);
+        var personRepository = new InMemoryPersonRepository(TestDomain.TestPersons[0].Id, storage);
+        var allEntries = personRepository.GetAll();
+        Assert.That(allEntries.Count, Is.EqualTo(3));
+        Assert.That(allEntries[0].Id, Is.EqualTo(TestDomain.TestPersons[0].Id));
+        Assert.That(allEntries[1].Id, Is.EqualTo(TestDomain.TestPersons[1].Id));
+        Assert.That(allEntries[2].Id, Is.EqualTo(TestDomain.TestPersons[2].Id));
+    }
+
+    [Test]
+    public void Add_NoEntriesInDictionary_EntriesSuccessfullyAdded()
+    {
+        var storage = new Dictionary<Guid, Person>();
+        var personRepository = new InMemoryPersonRepository(TestDomain.TestPersons[0].Id, storage);
+        personRepository.Add(TestDomain.TestPersons[0]);
+        personRepository.Add(TestDomain.TestPersons[1]);
+        personRepository.Add(TestDomain.TestPersons[2]);
+        Assert.That(personRepository.GetById(TestDomain.TestPersons[0].Id), Is.EqualTo(TestDomain.TestPersons[0]));
+        Assert.That(personRepository.GetById(TestDomain.TestPersons[1].Id), Is.EqualTo(TestDomain.TestPersons[1]));
+        Assert.That(personRepository.GetById(TestDomain.TestPersons[2].Id), Is.EqualTo(TestDomain.TestPersons[2]));
+    }
+
+    [Test]
+    public void GetCurrentUser_IsInDictionary_ReturnsCurrentUser()
+    {
+        Person defaultUser = new Person(Guid.NewGuid(), "DefaultFirstName", "DefaultLastName");
+
+        var personRepository = new InMemoryPersonRepository(defaultUser.Id);
+        personRepository.Add(defaultUser);
+
+        Assert.That(personRepository.GetCurrentUser(),
+            Is.EqualTo(defaultUser)); // in this case reference equals is fine because its the same object
+    }
+
+    [Test]
+    public void GetCurrentUser_IsNotInDictionary_ThrowsException()
+    {
+        Person defaultUser = new Person(Guid.NewGuid(), "DefaultFirstName", "DefaultLastName");
+
+        var personRepository = new InMemoryPersonRepository(defaultUser.Id);
+
+        Assert.That(() => personRepository.GetCurrentUser(),
+            Throws.TypeOf<ArgumentException>().With.Message
+                  .EqualTo(
+                      "default User is not in the dictionary")); // in this case reference equals is fine because its the same object
```

Comment:
I don't think that the comments add any value and should be removed.

When splitting a method call into multiple lines I would suggest moving each paramter on their own line:

```suggestion
        Assert.That(
            () => personRepository.GetCurrentUser(),
            Throws.TypeOf<ArgumentException>().With.Message
                  .EqualTo(
                      "default User is not in the dictionary")); // in this case reference equals is fine because its the same object
```